includes/admin/class-wp-job-manager-promoted-jobs-admin.php

```diff
@@ -212,6 +227,11 @@ public function promoted_jobs_custom_columns( $column ) {
 		if ( 'promoted_jobs' !== $column ) {
 			return;
 		}
+
+		if ( ! $this->can_promote_job( $post->ID ) ) {
+			return;
```

Comment:
I'm not sure but...maybe we can return here *why* the user can't promote that job? Just an idea. :thinking:

Reply:
I think it is outside of this PR's scope, but we could check with Andrei and add later. I feel like it will get quite messy repeated on the listing, but he might have a solution.